### ISSUE TITLE
Keyboard navigation of list in a FocusZone with an OverflowSet for each list item does not work

### ISSUE THREAD

**Issue by shahzebasif** [microsoft/fluentui#15533]
### Describe the issue:

We have a list of rows of buttons and labels. The entire list is in a FocusZone. Each item in the list also contains an OverflowSet. The OverflowSet will only be visible if the list item is hovered or focused.

The issue is that when the keyboard is used to click open the more items button, the callout opens for a brief moment before closing again. It doesn't stay open. There is no issue when using the mouse to navigate.

### Please provide a reproduction of the issue in a codepen:

The code sample below is basically what we have in the actual code. Except it has one problem not present in the actual code. I can't get the keyboard to navigate right into the Outer button and the OverflowSet. This is not the issue we see in our actual code but possibly another bug with how FocusZone's work?

That said, the code should still help illustrate how the actual code is structured.

https://codepen.io/shahzebasif/pen/Bazjepg?editors=0110

#### Actual behavior:

OverflowSet callout opens briefly and closes.

#### Expected behavior:

OverflowSet stays open and the keyboard can be used to navigate through the overflow items.

**Comment by paulgildea** [comment]
@shahzebasif Thanks for filing an issue. If there is another separate bug, could you please file another issue so that we can track them independently.

@tomi-msft - Try to see if you can come up with a root cause for the initial issue outlined. If not try to coordinate with @shahzebasif on a working repo that demonstrates the actual issue.

**Comment by shahzebasif** [comment]
Turns out it was an issue with the way we were doing conditional rendering which was causing this. The fix was to switch over to applying a style of `display: "none"` on the buttons which worked as we wanted it to.
